patterns/byod.md

```diff
@@ -0,0 +1,579 @@
+
+### Virtual desktop (BYOD Option 2)
+
+The Virtual desktop (Azure Virtual Desktop) device access option will leverage:
+
+* Azure AD MFA and Conditional Access policies
+* Secure Azure compute boundary that will provide the environment where data is accessed, creating full separation between the personal device and the corporate desktop.
+
+The outcome will allow the end user to: 
+
+* Access all M365 applications, from within the virtual desktop only
+* Access all other corporate applications, from within the virtual desktop only
+* Prevent copy, paste, saving of files and printing on the Mobile Device.
+
+The recommendation to implement this option is to utilise the [Azure Virtual Desktop Blueprint](https://desktop.gov.au/patterns/avd.html) pattern. The solution requires architectural decisions to be made that are outside the scope of this pattern. This BYOD option is more difficult to implement than the MAM BYOD option 1, but offers a less complex implementation than other virtual desktop solutions.
+
+The pattern recommends the following:
+
+* A suitably configured Azure subscription is available, which may include connectivity to the Agencies IT premises (e.g. data centre) where connectivity to hybrid Active Directory or corporate apps is required.
+* Windows Desktop OS multi-session operation system is used to offer greater supportability and cost savings than that of a single instance virtual machines.
+* Conditional Access policies are configured to allow access to the AVD application without enrollment or restriction, and to allow access to Microsoft 365 applications inside the virtual desktop. This will allow personal devices to connect to the service without restriction but only the virtual desktop to access the data.
+* Copy and paste actions, file copy and print are prevented within RDP client properties and group policy to prevent virtual channels being used to copy data in and out of the PROTECTED boundary (the virtual desktop).
+* Multifactor authentication is enforced with session controls to prevent multiple day use of the MFA token.
+* Identity Protection is configured to allow detection of risky sign-ins.
+
+**Mobile devices** or **personal devices** in the context of this BYOD option are consumer desktop devices using MacOS or Windows, or devices provided by another government Agency (cross-Agency working use-cases).
+
+#### Data segregation
+
+AVD allows users to access to Office 365 applications that are hosted on a Windows Virtual Desktop, including any hybrid applications provided these are configured appropriately within the Azure environment. These applications are running within a virtual desktop so the user operating the client device can only interact with the application inside the virtual machine, but cannot extract or upload data to and from the local endpoint when configured appropriately. 
```

Comment:
Replace `AVD allows users to access to Office` with `AVD allows users to access Office`

Maybe replace the following:

`These applications are running within a virtual desktop so the user operating the client device can only interact with the application inside the virtual machine, but cannot extract or upload data to and from the local endpoint when configured appropriately.`

with

`The user interacts with the applications within the virtual desktop preventing the user from extracting corporate data to the local physical device.`